src/main/java/com/hubspot/jinjava/tree/ExpressionNode.java

```diff
@@ -48,10 +49,23 @@ public OutputNode render(JinjavaInterpreter interpreter) {
 
     String result = Objects.toString(var, "");
 
+    TokenScannerSymbols symbols = interpreter.getConfig().getTokenScannerSymbols();
+    String expressionBegins = new StringBuilder()
```

Comment:
I didn't quite get what you're trying to say. Can you please explain a little bit so that I can make the changes?

Reply:
Sure. Add `getExpressionStart()` and `getExpressionEnd()` methods to `TokenScannerSymbols` and ensure they're only built once, rather than every time a node is rendered.